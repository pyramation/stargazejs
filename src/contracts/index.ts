import * as minter from './MinterContract';
import * as minterFromPartial from './MinterContract.from-partial';
import * as sg721 from './Sg721Contract';
import * as sg721FromPartial from './Sg721Contract.from-partial';

export const contracts = {
    Minter: {
        ...minter,
        ...minterFromPartial
    },
    Sg721: {
        ...sg721,
        ...sg721FromPartial
    }
};
