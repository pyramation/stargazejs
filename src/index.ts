export * from './proto';
export * from './contracts';