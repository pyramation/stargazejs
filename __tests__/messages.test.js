import { stargaze } from '../src/proto';

it('messages', async () => {
  expect(
    stargaze.mint.v1beta1.Minter.fromPartial({
      annualProvisions: 'str'
    })
  ).toMatchSnapshot();
});
