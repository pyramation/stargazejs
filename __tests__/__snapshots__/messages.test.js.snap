// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`messages 1`] = `
Object {
  "annualProvisions": "str",
}
`;
